app/utils/code-mirror-gutter-rs.ts

```
@@ -0,0 +1,532 @@
+    this.dom = document.createElement("div")
+    this.dom.className = "cm-gutterElement"
+    this.update(view, height, above, markers)
+  }
+
+  update(view: EditorView, height: number, above: number, markers: readonly GutterMarker[]) {
+    if (this.height != height) {
+      this.height = height
+      this.dom.style.height = height + "px"
+    }
+    if (this.above != above)
+      this.dom.style.marginTop = (this.above = above) ? above + "px" : ""
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)